Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -74,11 +74,11 @@ private void AddNewlyReleasedVersionToClosedIssuesOnlyAssociatedWithFullVersion
       var allJiraVersionsStartingWithFullVersion = _jiraVersionCreator.FindAllVersionsStartingWithVersionNumber(currentFullVersion);
 
       if (allJiraVersionsStartingWithFullVersion.Count == 0)
-        throw new InvalidOperationException($"Could not find versions starting with '{currentFullVersion}'.");
+        throw new UserInteractionException($"Could not find versions starting with '{currentFullVersion}' in JIRA.");
 
-      var currentFullJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(currentFullVersion)
-                                       ?? throw new InvalidOperationException(
-                                           $"Could not find any version with version number '{currentFullVersion}'.");
+      var currentFullJiraVersion =
+          _jiraVersionCreator.FindVersionWithVersionNumber(currentFullVersion) ??
+          throw new UserInteractionException($"Version '{currentFullVersion}' does not exist in JIRA.");
```

Comment:
```suggestion
      var currentFullJiraVersion =
          _jiraVersionCreator.FindVersionWithVersionNumber(currentFullVersion)
          ?? throw new UserInteractionException($"Version '{currentFullVersion}' does not exist in JIRA.");
```
Operators (eg boolean (&& and II,...), ??) are always at the line start, not line end. That way, you see what you intend to do when skimming the lines.